### PR TITLE
try to fix update-yaml.sh

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: manual-install/update-yaml.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "next
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-domaincheck"))')"
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-borgbackup"))')"
 OUTPUT="$(echo "$OUTPUT" | jq 'del(.services[] | select(.container_name == "nextcloud-aio-docker-socket-proxy"))')"
-OUTPUT="$(echo "$OUTPUT" | jq '.services[].depends_on |= if contains(["nextcloud-aio-docker-socket-proxy"]) then del(.[index("nextcloud-aio-docker-socket-proxy")])')"
+OUTPUT="$(echo "$OUTPUT" | jq '.services[].depends_on |=  if contains(["nextcloud-aio-docker-socket-proxy"]) then del(.[index("nextcloud-aio-docker-socket-proxy")])')"
```
Comment (szaimen) [inline manual-install/update-yaml.sh:24]: @Zoey2936 do you know how to fix this logic? This somehow still does not seem to work...

Comment (Zoey2936) [reply manual-install/update-yaml.sh:24]: I think there is a "else ." missing

Comment (szaimen) [reply manual-install/update-yaml.sh:24]: Oh it requires an else? 🤦‍♂️

Comment (Zoey2936) [reply manual-install/update-yaml.sh:24]: yes

Comment (Zoey2936) [reply manual-install/update-yaml.sh:24]: and an "end", so it needs to end with "else . end'"

Comment (szaimen) [reply manual-install/update-yaml.sh]: let me try this. thanks!

Comment (szaimen) [reply manual-install/update-yaml.sh]: Still seems to fail :/

image

Comment (szaimen) [reply manual-install/update-yaml.sh]: I need to probably first check if depends_on is there and then check if it contains something?

**Comment by szaimen** [comment]
Fixed it :)

**Pull request closed without merging**